<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pPrinterWidget</class>
 <widget class="QFrame" name="pPrinterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,10">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QGroupBox" name="gb_controls">
       <property name="title">
        <string>Controles</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QToolButton" name="connectBtn">
          <property name="text">
           <string>Connect</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btn_home">
          <property name="text">
           <string>Home</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_3">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_4">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gb_temps">
       <property name="title">
        <string>Temperaturas</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLCDNumber" name="lcdNumber"/>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumber_2"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gb_term">
       <property name="title">
        <string>terminal</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTextBrowser" name="termOut"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="termIn">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="sendCommand">
            <property name="text">
             <string>Send</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_view">
     <property name="title">
      <string>Vista Cama</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>termIn</sender>
   <signal>returnPressed()</signal>
   <receiver>sendCommand</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>383</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>582</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
